.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - run: mix local.rebar --force && mix local.hex --force
       - restore_cache:
           key: v1-mix-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
-      - run: mix do credo, format --check-formatted --dry-run
+      - run: mix do deps.get, credo, format --check-formatted --dry-run
```

Comment:
hmm this is strange, I was under the impression that https://github.com/omisego/ex_plasma/pull/59/files#diff-1d37e48f9ceff6d8030570cd36286a61L15 would cache the deps.